authentik/stages/email/stage.py

```diff
@@ -74,7 +75,7 @@ def get_token(self) -> FlowToken:
         pending_user = self.get_pending_user()
         current_stage: EmailStage = self.executor.current_stage
         valid_delta = timedelta(
```

Comment:
```suggestion
        valid_delta = timedelta_from_string(current_stage.token_expiry) + timedelta(minutes=1)  # + 1 because django timesince always rounds down
```

I think thats a bit easier to read